jobs/cam/plots/JEVS_CAM_PLOTS

```diff
@@ -65,14 +65,6 @@ else
     export VDATE=$PDYm1
 fi
 
-if [ $MODELNAME = nam_firewxnest ]
-then
- setpdy.sh 31
- . $DATA/PDY
- export VDATE=${VDATE:-$PDYm1}
```

Comment:
I see how you have incorporated the if/else logic for COMPONENT and MODELNAME to be inclusive for nam_firewxnest. However, I think you need to retain the definition for VDATE to be able to bring in an upstream definition. Please update line 65 of `JEVS_CAM_PLOTS` to look like this:
`export VDATE=${VDATE:-$PDYm1}`